arch/arm/boot/dts/tegra30-fujitsu-m532.dts

```diff
@@ -0,0 +1,1462 @@
+		stdout-path = "display0";
+
+		framebuffer@abc01000 {
+			compatible = "simple-framebuffer";
+			reg = <0xabc01000 0x3e8a00>;
+			width = <1280>;
+			height = <800>;
+			stride = <2560>;
+			format = "r5g6b5";
+			display = <&lcd>;
+			clocks = <&tegra_car TEGRA30_CLK_DISP1>,
+				 <&tegra_car TEGRA30_CLK_PWM>;
+			panel-supply = <&vdd_pnl1_reg>;
+			backlight-supply = <&vdd_bl_reg>;
+		};
+	};
+
+	memory@80000000 {
+		reg = <0x80000000 0x40000000>;
+	};
+	ram-console {
+		compatible = "android,ram-console";
+		reg = <0xbeb00000 0x100000>;
+	};
+
+	firmware {
+		trusted-foundations {
+			compatible = "tlm,trusted-foundations";
+			tlm,version-major = <2>;
+			tlm,version-minor = <8>;
+		};
+	};
+
+	reserved-memory {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+
+		firmware@bfe00000 {
+			// used by TF firmware
+			reg = <0xbfe00000 0x200000>;
+			no-map;
+		};
+
+		ramoops@beb00000 {
+			compatible = "ramoops";
+			reg = <0xbeb00000 0x100000>;
+			record-size = <0x00020000>;
+			console-size = <0x00040000>;
+			ftrace-size = <0x00040000>;
+			ecc-size = <16>;
+		};
+
+		lp0_vec@bddf9000 {
+			reg = <0xbddf9000 0x2000>;	// passed from bootloader (ATAGS/NVIDIA, cmdline)
+		};
+/*
+		reserved@bddf6000 {
+			reg = <0xbddf6000 0x2000>;	// passed from bootloader (ATAGS/NVIDIA)
+		};
+*/
+		framebuffer@abc01000 {
+			reg = <0xabc01000 0x3e9000>;
+			no-map;
+		};
+	};
+
+	cpus {
+		cpu0: cpu@0 {
+			cpu-supply = <&vddctrl_reg>;
+			operating-points-v2 = <&cpu0_opp_table>;
+			#cooling-cells = <2>;
+		};
+		cpu@1 {
+			cpu-supply = <&vddctrl_reg>;
+			operating-points-v2 = <&cpu0_opp_table>;
+		};
+		cpu@2 {
+			cpu-supply = <&vddctrl_reg>;
+			operating-points-v2 = <&cpu0_opp_table>;
+		};
+		cpu@3 {
+			cpu-supply = <&vddctrl_reg>;
+			operating-points-v2 = <&cpu0_opp_table>;
+		};
+	};
+
+	thermal-zones {
+		local {
+			polling-delay-passive = <1000>; /* milliseconds */
+			polling-delay = <5000>; /* milliseconds */
+
+			thermal-sensors = <&temp 0>;
+
+			trips {
+				trip0: cpu-alert0 {
+					/* start throttling at 50C */
+					temperature = <50000>;
+					hysteresis = <3000>;
+					type = "passive";
+				};
+
+				trip1: cpu-crit {
+					/* shut down at 60C */
+					temperature = <60000>;
+					hysteresis = <2000>;
+					type = "critical";
+				};
+			};
+
+			cooling-maps {
+				map0 {
+					trip = <&trip0>;
+					cooling-device = <&cpu0 THERMAL_NO_LIMIT THERMAL_NO_LIMIT>;
+				};
+			};
+		};
+	};
+
+	panel: panel {
+		compatible = "innolux,n101icg-l21", "innolux,g101ice-l01", "simple-panel";
+
+		power-supply = <&vdd_pnl1_reg>;
+		enable-gpios = <&gpio TEGRA_GPIO(N, 6) GPIO_ACTIVE_HIGH>;
+
+		backlight = <&backlight>;
+	};
+
+	backlight: backlight {
+		compatible = "pwm-backlight";
+
+		enable-gpios = <&gpio TEGRA_GPIO(H, 2) GPIO_ACTIVE_HIGH>;
+		power-supply = <&vdd_bl_reg>;
+		pwms = <&pwm 0 5000000>;
+
+		brightness-levels = <2 4 8 16 32 64 128 255>;
+		default-brightness-level = <6>;
+	};
+
+	sound {
```

Comment:
check if in chagall sources is also this codec (wm8903) used.

Reply:
Yes, WM8903